tests/checks/mock/test_dns_check.py

```diff
@@ -52,6 +55,18 @@ def test_success(self, mocked_query):
                                 tags=['resolved_hostname:www.example.org', 'nameserver:127.0.0.1'])
         self.coverage_report()
 
+    @mock.patch.object(Resolver, 'query', side_effect=success_query_mock)
+    def test_success_CNAME(self, mocked_query):
+        config = {
+            'instances': [{'hostname': 'www.example.org', 'nameserver': '127.0.0.1', 'record_type': 'CNAME'}]
```

Comment:
we do it in the `test_invalid_config` test (https://github.com/DataDog/dd-agent/pull/2653/files#diff-3254c25e4da8782409c2288e998ebc38R96)